app/views/About.js

```diff
@@ -32,108 +30,69 @@ export const AboutScreen = ({ navigation }) => {
       <ScrollView
         contentContainerStyle={styles.contentContainer}
         alwaysBounceVertical={false}>
-        <View style={styles.spacer} />
-        <View style={styles.spacer} />
 
-        <Typography use='headline2'>{aboutHeader}</Typography>
-        <View style={{ height: 10 }} />
+        <Typography use='headline2' style={styles.heading}>{aboutHeader}</Typography>
         <Typography use='body2'>
           {t('label.about_para')}
-          {/* Space between the copy & link*/}
-          <Text> </Text>
           <Typography
             style={styles.hyperlink}
             onPress={() => {
               Linking.openURL('https://covidsafepaths.org/');
-            }}>
-            {/* eslint-disable-next-line react-native/no-raw-text */}
-            {'covidsafepaths.org'}
+            }}>{"\n"}
```

Comment:
If these are different lines now, can we separate them by de-nesting the `Typography` components instead?

Reply:
Yeah that's better - will do